xml/System.Buffers/IBufferWriter`1.xml

```diff
@@ -15,8 +15,8 @@
   </TypeParameters>
   <Interfaces />
   <Docs>
-    <typeparam name="T">To be added.</typeparam>
-    <summary>To be added.</summary>
+    <typeparam name="T">The type of the items in the <see cref="T:System.Buffers.ReadOnlySequence`1" />.</typeparam>
```

Comment:
This shouldn't be `ReadOnlySequence`. Where did this come from?

Reply:
Is there a way for me to read/review the new docs as an md/etc. and not as an xml?

The docs don't show up yet:
https://docs.microsoft.com/en-us/dotnet/api/system.buffers.ibufferwriter-1?view=netcore-2.2

Reply:
The production build is still running.

You can see on review: https://review.docs.microsoft.com/en-us/dotnet/api/system.buffers.ibufferwriter-1?view=netcore-2.2&branch=master

We've made some edits on top of the /// comments. I'll change that to IBuffer.

The /// comments was a stop gap to add some docs here.